### PR TITLE
[ChakraCore] enable find_package

### PR DESCRIPTION
**Pull request by Pospelove**

### CODE REVIEW

**Review by LilyWangLL — DISMISSED** [review]

File: ports/chakracore/unofficial-chakracore-config.cmake [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  INTERFACE_INCLUDE_DIRECTORIES "${ChakraCore_INCLUDE_DIR}"
+)
+
+find_library(ChakraCore_LIBRARY_RELEASE NAMES ChakraCore PATHS "${CMAKE_CURRENT_LIST_DIR}/../../lib" NO_DEFAULT_PATH REQUIRED)
+find_library(ChakraCore_LIBRARY_DEBUG NAMES ChakraCore PATHS "${CMAKE_CURRENT_LIST_DIR}/../../debug/lib" NO_DEFAULT_PATH REQUIRED)
+
+set_target_properties(unofficial::chakracore::chakracore PROPERTIES
+  IMPORTED_LOCATION_DEBUG "${ChakraCore_LIBRARY_DEBUG}"
+  IMPORTED_LOCATION_RELEASE "${ChakraCore_LIBRARY_RELEASE}"
+)
```
Comment (dg0yt) [inline ports/chakracore/unofficial-chakracore-config.cmake:15]: You cannot use `IMPORTED_LOCATION` properties on an `IMPORT` library. Please change the library type to `UNKNOWN`.

**Review by dg0yt — CHANGES_REQUESTED** [review]
Please validate the usage before approval.

File: ports/chakracore/unofficial-chakracore-config.cmake (hunk 1 above)
Comment (dg0yt) [reply ports/chakracore/unofficial-chakracore-config.cmake:15]: ~IMPORT~INTERFACE

Comment (dg0yt) [reply ports/chakracore/unofficial-chakracore-config.cmake:15]: It is also necessary to set `IMPORTED_CONFIGURATIONS`.

**Comment by LilyWangLL** [comment]
> Please validate the usage before approval.

`usage` test pass on x64-windows.

File: ports/chakracore/unofficial-chakracore-config.cmake (hunk 1 above)
Comment (LilyWangLL) [reply ports/chakracore/unofficial-chakracore-config.cmake:15]: > You cannot use `IMPORTED_LOCATION` properties on an `INTERFACE` library. Please change the library type to `UNKNOWN`.

Ping @Pospelove for response, could you please apply this suggestion?

**Comment by Pospelove** [comment]
@dg0yt @LilyWangLL done

plus added `if(TARGET)` check in case of multiple calls to `find_package` on user side

tested with project that uses chakracore port. pr: https://github.com/skyrim-multiplayer/skymp/pull/1304

**Comment by Pospelove** [comment]
@LilyWangLL

**Comment by Pospelove** [comment]
bump
@LilyWangLL

**Review by LilyWangLL — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for the improvement!

**Pull request merged**